ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/LinearRao.java

```diff
@@ -52,27 +59,68 @@ public String getVersion() {
         return "1.0.0";
     }
 
+    private SystematicSensitivityAnalysisResult runSensiFallback(Network network,
+                                                                 Crac crac,
+                                                                 ComputationManager computationManager,
+                                                                 LinearRaoParameters parameters) {
+        try { // with default parameters
+            return runSensi(network, crac, computationManager, parameters);
+        } catch (SensitivityComputationException e1) {
+            activeSensiParameters = ActiveSensiParameters.FALLBACK;
+            try { // with fallback parameters
+                return runSensi(network, crac, computationManager, parameters);
+            } catch (SensitivityComputationException e2) {
+                throw new SensitivityComputationException("Sensitivity computation failed with all available sensitivity parameters.");
+            }
+        }
+    }
+
+    private SystematicSensitivityAnalysisResult runSensi(Network network,
+                                                         Crac crac,
+                                                         ComputationManager computationManager,
+                                                         LinearRaoParameters parameters) {
+
+        // Run sensitivity analysis
+        SystematicSensitivityAnalysisResult sensitivityAnalysisResult = SystematicSensitivityAnalysisService.runAnalysis(network, crac, computationManager, selectSensiParameters(parameters));
```

Comment:
To separate complexity levels I wouldn't have put selectSensiParameters here. So runSensi just manage a simple sensi with fixed parameters.